src/css/base/_overlay.scss

```diff
@@ -16,6 +16,6 @@
 }
 .coa-overlay__content {
   margin: 0 auto;
-  max-width: $coa-textwrap-width;
+  max-width: $coa-wrapper-sm-maxwidth;
```

Comment:
this is a confusing variable name

Reply:
any other suggestions? it's basically the max-width for the smaller wrapper. It seems like we have two maxwidths in Celine's designs -- 1325 and 720.

$coa-wrapper-maxwidth: 1325px;
$coa-wrapper-sm-maxwidth: 720px;